SPECS/dhcp/dhcp.spec

```diff
@@ -38,7 +41,15 @@ The ISC DHCP Client, dhclient, provides a means for configuring one or more netw
 
 
 %prep
-%autosetup -p1
+%setup -n dhcp-%{version}
+pushd bind
+tar -xf bind.tar.gz
+pushd bind-9*
+%patch0 -p1
+%patch1 -p1
+%patch2 -p1
+popd
+popd
```

Comment:
A few suggestions to make the `%prep` more automated and easier to update with future patches:
- Let's use `%autopatch` but modify the paths in the 3 new patch files, so that they modify the `bind` directory.
- Reduce a bit of clutter with `pushd` and `popd` to make the logic easier to follow.
- Add a symbolic link to the untarred `bind-9*` to make patching smoother over minor version changes.

This would require patch paths changes from:
```
--- a/lib/dns/opensslecdsa_link.c
```

to:

```
--- a/bind_ln/lib/dns/opensslecdsa_link.c
```

```suggestion
%autosetup

# Extracting bundled 'bind' to allow some of the patches to modify it.
tar -C bind -xf bind.tar.gz
ln -s bind/bind-9* bind_ln

%autopatch -p1
```

Reply:
%autosetup  automatically applies all patches listed in the spec, we need to apply after extract bind dir, so i'll keep `%setup -n dhcp-%{version}`  which solely unpack dhcp source